release/kubernetes-manifests.yaml

```diff
@@ -144,8 +144,6 @@ spec:
             value: "1"
           - name: DISABLE_PROFILER
             value: "1"
-          # - name: JAEGER_SERVICE_ADDR
```

Comment:
You don't want to manually commit changes to the `release` directory since that directory is linked to the latest release. In this case, you're making source code changes but those changes won't be part of the latest release (until a release happens). This file is automatically generated by https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/hack/make-release.sh

TL;DR: revert the changes to this file.

Reply:
Done